renderdoc/driver/gl/gl_hooks_egl.cpp

```diff
@@ -509,6 +505,12 @@ __attribute__((visibility("default"))) EGLBoolean eglSwapBuffers(EGLDisplay dpy,
   eglhooks.eglQuerySurface_real(dpy, surface, EGL_HEIGHT, &height);
   eglhooks.eglQuerySurface_real(dpy, surface, EGL_WIDTH, &width);
 
+  GLInitParams &init = eglhooks.GetDriver()->GetInitParams();
+  int colorspace;
+  eglhooks.eglQuerySurface_real(dpy, surface, EGL_COLORSPACE, &colorspace);
+  // GL_SRGB8_ALPHA8 is specified as color-renderable, unlike GL_SRGB8.
+  init.isSRGB = init.colorBits == 32 && colorspace == EGL_COLORSPACE_sRGB;
```

Comment:
based on the egl.h/docs the `EGL_COLORSPACE` and `EGL_COLORSPACE_sRGB` constant are related to OpenVG and not to GL ES. We would need to use `EGL_GL_COLORSPACE` and `EGL_GL_COLORSPACE_sRGB` but that is only available from EGL 1.5 or via the EGL_KHR_gl_colorspace extension.